src/cascadia/ut_app/JsonUtilsTests.cpp

```diff
@@ -295,17 +295,30 @@ namespace TerminalAppUnitTests
     template<typename TExpected, typename TJson>
     static void TryBasicType(TExpected&& expected, TJson&& json)
     {
+        // test FromJson
         Json::Value jsonObject{ json };
         const auto value{ GetValue<TExpected>(jsonObject) };
         VERIFY_ARE_EQUAL(expected, value, NoThrowString{}.Format(L"(type: %hs)", typeid(TExpected).name()));
+
+        // test ToJson
+        {
+            const std::string key{ "myKey" };
+
+            Json::Value expectedJson{};
+            expectedJson[key] = json;
+
+            jsonObject = {};
+            SetValueForKey<TExpected>(jsonObject, key, expected);
+            VERIFY_ARE_EQUAL(expectedJson, jsonObject);
+        }
     }
 
     void JsonUtilsTests::BasicTypeConversion()
     {
         // Battery of all basic types ;P
         TryBasicType(std::string{ "hello" }, "hello");
         TryBasicType(int{ -1024 }, -1024);
-        TryBasicType(unsigned int{ 1024 }, 1024);
+        TryBasicType(std::numeric_limits<unsigned int>::max(), std::numeric_limits<unsigned int>::max());

```

Comment:
why?

Reply:
Since I `VERIFY_ARE_EQUAL` the json objects, `ToJson`-ing a number (even though it's marked as an unsigned int), is recorded as an int, if it fits. So what doesn't fit in an int...well, this is how we got here hahaha.

Reply:
I don't love that ;P but okay